### DISCUSSION TITLE
How to check the prompt sent to the LLM when we uses the ChatCompletion.create() method ?

### DISCUSSION THREAD

**Discussion by AntoninLeroy** [vllm-project/vllm#2989] [Q&A]
![image](https://github.com/vllm-project/vllm/assets/42861883/61f4639c-a6de-439a-bfce-848705796ea6)

I use Mixtral8x7b and I want to make sure that the prompt template is correct.

Is it possible ?

**Comment by timeconnection** [comment]
you can see the input in vllm logs.
![Uploading vllm_log.png…]()
